### ISSUE TITLE
MD - Error logout on device

### ISSUE THREAD

**Issue by tiagoandrepro** [ookamiiixd/baileys-api#11]
When disconnecting the device the API falls, the same error does not happen with legacy

```
Error: Connection Closed
    at sendRawMessage (/Users/tiagoandre/Projetos/Sites/baileys-api/node_modules/@adiwajshing/baileys/lib/Socket/socket.js:63:19)
    at sendNode (/Users/tiagoandre/Projetos/Sites/baileys-api/node_modules/@adiwajshing/baileys/lib/Socket/socket.js:71:16)
    at sendReceipt (/Users/tiagoandre/Projetos/Sites/baileys-api/node_modules/@adiwajshing/baileys/lib/Socket/messages-send.js:99:15)
    at Object.sendReadReceipt (/Users/tiagoandre/Projetos/Sites/baileys-api/node_modules/@adiwajshing/baileys/lib/Socket/messages-send.js:105:16)
    at async EventEmitter.<anonymous> (file:///Users/tiagoandre/Projetos/Sites/baileys-api/whatsapp.js:161:17)
Running cleanup before exit.
```

**Comment by ookamiiixd** [comment]
Yes, i already put a note for this in the [readme](https://github.com/ookamiiixd/baileys-api#known-issue). I don't find any solution yet for this, still waiting for response from the baileys author.

**Comment by ookamiiixd** [comment]
This issue should be fixed on Baileys version **4.1.0**, closing this.
